server/js/apexsignature.js

```diff
@@ -32,21 +32,23 @@ var apexSignature = {
         return base64;
     },
     // save to DB function
-    save2Db: function(pAjaxIdentifier, pRegionId, pImg, callback) {
+    save2Db: function(pOptions, pRegionId, pImg, callback) {
         // img DataURI to base64
         var base64 = apexSignature.dataURI2base64(pImg);
         // split base64 clob string to f01 array length 30k
         var f01Array = [];
         f01Array = apexSignature.clob2Array(base64, 30000, f01Array);
         // Apex Ajax Call
-        apex.server.plugin(pAjaxIdentifier, {
-            f01: f01Array
+        apex.server.plugin(pOptions.ajaxIdentifier, {
+            f01: f01Array,
+            // #13: Allows for items to be submitted
+            pageItems: pOptions.ajaxItemsToSubmit.split(',')
```

Comment:
There will be a *null pointer issue* that when we left `Page Items to Submit` empty

Reply:
hello, I have problems with page items to submit.
It seems there are no problems with some items in a region above the plugin, because it functions.
I also heve 2 item in a region below the plugin and the value in the PL/SQL code of the plugis is null, or seems to have the preceeding value.

the correct item are hidden and already valued when the page arrives the first time, becouse this page comes from "link to another page"
the 2 ko items are normal page items, and the value I want to insert in the Db is what typed by the user.

Can you suggest something?
Thank you, Angelo

Reply:
Sorry, applied all integrations and now it functions well, doesn't matter in which region the items are positioned.
Thank you,

Angelo